src/selectors/producers.js

```diff
@@ -0,0 +1,29 @@
+import _ from 'underscore';
+import { createSelector } from 'reselect';
+
+function selectSuppliersWithProducer(suppliers, producers) {
+  const entities = _.map(suppliers, (supplier) => {
+    const producer = _.omit(producers, ['created_at', 'updated_at', 'id']);
+
+    return {
+      ...producer,
+      ...supplier,
+    };
```

Comment:
mh... nice question. This selector actually doesn't work... :grimacing: 

I need to debug it but for now it just breaks the app :boom: